### ISSUE TITLE
Can get values greater than 255 from color_rgb_bytes.

### ISSUE THREAD

**Issue by caternuson** [adafruit/Adafruit_CircuitPython_TCS34725#10]
Doc string claims the return should be 3-tuple with range 0-255. But it's possible to get values greater than this.
```python
Adafruit CircuitPython 3.0.0 on 2018-07-09; Adafruit CircuitPlayground Express with Crickit libraries with samd21g18
>>> import board, busio
>>> import adafruit_tcs34725
>>> i2c = busio.I2C(board.SCL, board.SDA)
>>> sensor = adafruit_tcs34725.TCS34725(i2c)
>>> sensor.integration_time
2.4
>>> sensor.gain
1
>>> sensor.color_rgb_bytes
(18, 25, 9)
>>> sensor.integration_time = 50
>>> sensor.integration_time
48.0
>>> sensor.color_rgb_bytes
(2, 554, 570)
>>> 
```

**Comment by jerryneedell** [comment]
Where did the formulae for the conversions come from - I can't find it in the data sheet of the arduino library.

**Comment by microbuilder** [comment]
> Where did the formulae for the conversions come from - I can't find it in the data sheet of the arduino library.

Not sure about this myself, but I'll look into it and if necessary reimplement the function.

**Comment by caternuson** [comment]
Thanks here as well. Not knowing what that was doing was keeping me from attempting any code updates. So thanks for taking a look.

**Comment by caternuson** [comment]
Fixed with #16
